x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/index.test.ts

```diff
@@ -41,9 +41,18 @@ describe('EndpointList store concerns', () => {
         total: 0,
         loading: false,
         error: undefined,
-        details: undefined,
-        detailsLoading: false,
-        detailsError: undefined,
+        endpointDetails: {
```

Comment:
👍 

I'm working Host Isolation from the endpoint list and will need a namespace here as well for it. I will create mine outside of this, but once you merge it, I will move it in here.

Reply:
sounds good 👍